toolkit/tools/precacher/precacher.go

```diff
@@ -350,12 +355,12 @@ func precachePackage(pkg *repocloner.RepoPackage, packagesAvailableFromRepos map
 	const (
 		downloadRetryAttempts = 2
 		downloadRetryDuration = time.Second
+		stripEpoch            = false
+		leaveEpoch            = true
```

Comment:
File names for RPM packages never contain the epoch number - that's only encoded in the RPMs metadata inside these files. We can just always strip the epoch from the version, if it's present and simplify the code a bit.

Reply:
Good to know, I've simplified this then, and just always strip the epoch if its present in the version